domains/qwertystars.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "qwertystars",
+        "email": "srijan.guchhait@gmail.com"
+    },
+    "record": {
+        "CNAME":"qwertystars.000.pe"
```

Comment:
Incomplete website I assume?

Reply:
yes wanting to reserve this domain for temp. will buy new ip address and host of my own in the next few months.

Reply:
Is it possible to do it or should i try again later?

Reply:
I don't think anyone is going to take this domain, You can make another PR again after a few months.